### DISCUSSION TITLE
Shows composer.json file is invalid though it is valid

### DISCUSSION THREAD

**Discussion by prashantbw** [enlightn/enlightn#131] [Q&A]
Hi,
  When I run enlightn, for composer.json check it shows failed.
  
```
  Your application's composer.json file is valid. Failed
Your application's composer.json file is not valid. Run the composer validate command to view more details.
```

If I check it with "composer validate" command it shows composer.json is valid.

I am on Laravel 8.  Is it false positive or am I missing something

**Comment by startork** [comment]
Not quite an answer but I am also getting this issue.
I think it may be because composer is not installed on my webserver (where we run the enlightn report) and so this check fails. We've decided to ignore this check as it is definitely not worth installing composer on the server for :)
